app/controllers/main_controller.rb

```diff
@@ -132,7 +132,7 @@ def init_popup_section
   def init_validation_section
 
     # let's lay this out using the grid!
-    rmq.append(UIView, :validation_section).tap do |q|
+    rmq.append(UIView, :validation_section) do |q|
```

Comment:
Figured I'd try using the new syntax in a couple random places in the tests, to make sure it works in real world situations. I have more direct tests below.